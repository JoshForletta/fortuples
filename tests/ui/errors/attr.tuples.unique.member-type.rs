use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::member_type(A)]
    #[tuples::member_type(B)]
    impl Container<#Tuple> {}
}

fn main() {}
