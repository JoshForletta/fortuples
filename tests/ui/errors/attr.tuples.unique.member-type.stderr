error: `member_type` should be set only once
 --> tests/ui/errors/attr.tuples.unique.member-type.rs:7:15
  |
7 |     #[tuples::member_type(B)]
  |               ^^^^^^^^^^^

error: the first definition was here
 --> tests/ui/errors/attr.tuples.unique.member-type.rs:6:15
  |
6 |     #[tuples::member_type(A)]
  |               ^^^^^^^^^^^
